multiqc/modules/kat/kat.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from KAT """
+import logging
+from collections import OrderedDict
+import json
+
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import table
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='KAT', anchor='kat',
+                                            href="https://github.com/TGAC/KAT",
+                                            info="is an toolkit for analysing sequencing data via its k-mer spectra.")
+
+        # Find and load any KAT dist analysis reports
+        self.kat_data = dict()
+        for c_file in self.find_log_files('kat'):
+            s_name = self.clean_s_name(c_file['s_name'][:-14], c_file['root'])
```

Comment:
It's better to use `.replace()` or the core config filename cleaning options instead of removing a set number of characters. Remember that users can override your search patterns, so you can't necessarily rely on the filenames being what you exist.